cloudflare/resource_cloudflare_load_balancer_pool_test.go

```diff
@@ -205,11 +207,19 @@ resource "cloudflare_load_balancer_pool" "%[1]s" {
     address = "1.1.1.2"
     enabled = false
     weight = 1.0
+    header {
```

Comment:
This test is currently failing as the hostname (random resource ID) is invalid. We need a real domain + IP combo here. I'm working on finding a couple we can use in order to get this test green again.